unit-tests/C/unit-tests-rsutil-c.c

```diff
@@ -0,0 +1,36 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#include <librealsense2/rsutil.h>
+#include <stdio.h>
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
Since this is supposed to be compiled in C only, I'd expect to see `#error` here.